webpack/scenes/ModuleStreams/Details/Profiles/ProfileRpmsCellFormatter.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+
+import './ProfileRpmsCellFormatter.scss';
+
+class ProfileRpmsCellFormatter extends Component {
+  constructor(props) {
+    super(props);
+    this.minAmount = 10;
+    this.state = {
+      expanded: false,
+      showAmount: this.minAmount,
+    };
+  }
+
+  onClick = () => {
+    const { rpms } = this.props;
+
+    this.setState(
+      { expanded: !this.state.expanded },
+      () => {
+        const showAmount = this.state.expanded ? rpms.length : this.minAmount;
+        this.setState({ showAmount });
+      },
+    );
+  };
+
+  iconName = () => (this.state.expanded ? 'angle-down' : 'angle-right');
+
+  render() {
+    const { rpms } = this.props;
+    const largeList = rpms.length > this.minAmount;
+
+    return (
+      <td>
+        {largeList && <Icon
+          className="expand-profile-rpms"
+          onClick={this.onClick}
+          name={this.iconName()}
+        />}
+        {rpms.map(rpm => rpm.name).slice(0, this.state.showAmount).join(', ')}
```

Comment:
I can clean up, but I'm not sure I understand your second point, do you have an example?

Reply:
instead of iterate over the entire list first ( map -> slice), iterate over only the relevant data (slice  -> map):
 ``` javascript
{
  rpms
    .slice(rpm => rpm.name)
    .map(0, this.state.showAmount)
    .join(', ');
}
```

Reply:
I'll try that, I think the first two arguments need to be switched, but I get your point now :) Do we have a prettier config that works with our eslint? I also like to use prettier, but have never gotten it working with katello